### DISCUSSION TITLE
Automatically Center New Windows

### DISCUSSION THREAD

**Discussion by Kha1r0** [rxhanson/Rectangle#1436] [Ideas]
Hey There!
The GNOME Desktop Environment on Linux has this little feature which i absolutely adore.
It's called "center-new-windows", and it does exactly that. When a new window is opened, it is automatically placed in the center of the current monitor. I haven't found any modern equivalent to this on macOS. Maybe it's something for rectangle?

![image](https://github.com/user-attachments/assets/618f1d7c-c7b4-4152-9144-3d25af1e57e1)
(Picture: dconf editor in ubuntu showing the "center-new-windows"-option)

**Comment by rxhanson** [comment]
Thanks for the feedback! The reason why this hasn't been implemented is because it would require polling the windows to see when a new one has been opened. It's not really something I have a desire to add to the app, since polling would have constant resource usage, and the behavior could be a little awkward if the polling interval is noticeably long. If anyone wants to implement it, I could see merging it in as a feature behind a terminal command configuration.
